media/js/firefox/hello/start.js

```diff
@@ -25,7 +28,32 @@
 
     // GA event for successful Hello conversations
     function trackGAConversationConnect() {
+        // get source from localStorage
+        try {
+            tourSource = localStorage.getItem('hello_ftu_tour_source');
+        } catch (ex) {
+            tourSource = 'none';
+        }
+        gaTrack(['_setCustomVar', 13, 'Hello FTU Referral', tourSource, 2]);
         gaTrack(['_trackEvent', '/hello/start interactions', 'tour', 'TourConnectConversation']);
+        try {
+            localStorage.setItem('hello_ftu_tour_source', 'none');
+        } catch (ex) { }
+    }
+
+    // Save the tourSource to localStorage for use in conversation connect
+    function saveTourSourceToLocalStorage() {
+        try {
+            localStorage.setItem('hello_ftu_tour_source', tourSource);
+        } catch (ex) { }
+    }
+
+    // Get query string parameters
+    function getParameterByName(name) {
+        name = name.replace(/[\[]/, "\\[").replace(/[\]]/, "\\]");
```

Comment:
Teeny tiny nit: got some mixed quotes in this function. For consistency sake, let's stick to single quotes